Tests/SonarScanner.MSBuild.TFS.Test/SonarScanner.MSBuild.TFS.Test.csproj

```diff
@@ -1,8 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>net48</TargetFrameworks>
-    <LangVersion>8.0</LangVersion>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
Removed `GenerateAssemblyInfo`, it is already in `Directory.Build.targets` above